subgraphs/erc20/README.md

```diff
@@ -1 +1,27 @@
-# ERC20-Subgraph
\ No newline at end of file
+# ERC20 Token Subgraph
+The subgraph read erc20 token list from IPFS, monitor the tokens' event and aggregate the results accordingly.
+
+## Calculation Methodology v1.0.0
+
+### Token's Hold Count
+
+Count of Unique Addresses which have hold the token: 
+
+### Token's Transfer Count
+
+Count of Transfer Event which have interacted with the protocol via transfer transaction
+
+`Transfers`
+
+### Token's Cumulative Unique Users
+Count of Unique Addresses which have interacted with the protocol via any transaction
+
+`Transfers`
```

Comment:
what's this `transfer` for?

Reply:
change to Holder Count in the latest commit.